tests/integration/home/home_spec.ts

```diff
@@ -0,0 +1,74 @@
+import { loginAsAdmin } from '../../shared/login';
+
+describe('Home Tests',()=>{
+  
+    it('should appear disclaimer beacause no data protection officer are in system',()=>{
+        loginAsAdmin();
+        
+        cy.url().should('equal', Cypress.config().baseUrl + '/?from=login');
+        cy.get('#dataProtectionOfficeDisclaimer').should('have.text', 'Warnmeldung');
+        cy.get('#resolveButton').should('exist');
+        cy.get('#ignoreButton').should('exist');        
+    })
```

Comment:
Der Test schlägt grade fehl, weil vor dem Datenschutzbeauftragen-Dialog noch der Dialog zum Ändern des Admin-Passworts beim ersten Login angezeigt wird. Das Setzen des Passworts findet erst im Login-Test statt, der aber nach dem Home-Test ausgeführt wird (die Dateien werden alphabetisch abgearbeitet).

Ich würde entweder diesen Test nach login_spec.ts verschieben (hinter den "should login with or without an initial password change"-Test), oder wir benennen die Dateien um (z.B. 01_login_spec.ts und 02_home_spec.ts,...), damit die Reihenfolge passt. 
Was meinst du?